core/templates/services/assets-backend-api.service.spec.ts

```diff
@@ -212,6 +212,58 @@ describe('Assets Backend API Service', function() {
       $rootScope.$apply();
     });
 
+    it('should successfully save a math Svg', function(done) {
+      var successMessage = 'Math SVG was successfully saved.';
+      // @ts-ignore in order to ignore JQuery properties that should
+      // be declarated.
+      spyOn($, 'ajax').and.callFake(function() {
```

Comment:
This I feel is a poor test for the code it is testing. Is there a way to ensure that the $.ajax call from the saveMathExpressionImage function actually creates the correct payload? All that is never actually asserted anywhere.

Reply:
done, I've added more detailed tests